src/main/java/uk/gov/hmcts/reform/unspec/helpers/LocalDateTimeHelper.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.unspec.helpers;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+
+public class LocalDateTimeHelper {
+
+    public static final ZoneId UTC_ZONE = ZoneId.of("UTC");
+    public static final ZoneId LOCAL_ZONE = ZoneId.of("Europe/London");
+
+    private LocalDateTimeHelper() {
+    }
+
+    @SuppressWarnings({"AbbreviationAsWordInName"})
```

Comment:
I think we should not SuppressWarnings in code if we don't find particular checkstyle rule useful. I'd say that `checkstyle.xml` should be updated instead. How about changing `allowedAbbreviationLength` to 3 there and remove the suppression?